app/javascript/dashboard/routes/dashboard/inbox/components/InboxCard.vue

```diff
@@ -10,42 +10,83 @@
   >
-    <div class="flex relative items-center justify-between w-full">
+    <div
+      class="flex flex-row justify-between relative items-center w-full gap-1.5"
+    >
       <div
         v-if="isUnread"
         class="absolute ltr:-left-3.5 rtl:-right-3.5 flex w-2 h-2 rounded bg-woot-500 dark:bg-woot-500"
       />
-      <InboxNameAndId :inbox="inbox" :conversation-id="primaryActor.id" />
-
-      <div class="flex gap-2">
-        <PriorityIcon :priority="primaryActor.priority" />
-        <StatusIcon :status="primaryActor.status" />
-      </div>
-    </div>
-
-    <div class="flex flex-row justify-between items-center w-full gap-2">
       <Thumbnail
         v-if="assigneeMeta"
         :src="assigneeMeta.thumbnail"
         :username="assigneeMeta.name"
         size="16px"
       />
       <span
-        class="flex-1 text-slate-800 dark:text-slate-50 text-sm overflow-hidden text-ellipsis whitespace-nowrap"
+        class="flex-1 text-slate-900 dark:text-white text-sm overflow-hidden text-ellipsis whitespace-nowrap"
         :class="isUnread ? 'font-medium' : 'font-normal'"
       >
         {{ pushTitle }}
       </span>
-      <span
-        class="font-medium text-slate-600 dark:text-slate-300 text-xs whitespace-nowrap"
-      >
-        {{ lastActivityAt }}
-      </span>
     </div>
-    <div v-if="snoozedUntilTime" class="flex items-center">
-      <span class="text-woot-500 dark:text-woot-500 text-xs font-medium">
-        {{ snoozedDisplayText }}
-      </span>
+
+    <div
+      ref="inboxCardInfo"
+      class="flex flex-row justify-between items-center w-full gap-2"
+    >
+      <div
+        v-if="snoozedUntilTime || hasLastSnoozed"
+        class="flex flex-row flex-1 gap-1 items-center min-w-0"
+      >
+        <fluent-icon
+          :icon="hasLastSnoozed ? 'snooze-timer' : 'sharp-timer'"
+          type="outline"
+          class="text-woot-500 dark:text-woot-500 flex-shrink-0"
+          size="16"
+        />
+        <span
+          class="text-woot-500 dark:text-woot-500 text-xs font-medium truncate"
+        >
+          {{ snoozedDisplayText }}
+        </span>
+      </div>
+      <div
+        v-else
+        class="flex items-center flex-row justify-start gap-2 min-w-0"
+      >
+        <inbox-card-info
+          :inbox="inbox"
+          :conversation-id="primaryActor.id"
+          :conversation-labels="primaryActor.labels"
+          :parent-element-width="inboxCardInfoElementWidth"
+        />
+        <div
+          v-show="notificationTypes"
+          class="flex flex-row items-center gap-0.5 w-fit min-w-0"
+        >
+          <fluent-icon
+            :icon="notificationTypeIcon"
+            type="outline"
+            class="text-woot-500 dark:text-woot-500 flex-shrink-0"
+            size="16"
+          />
+          <span
+            class="font-medium text-woot-500 dark:text-woot-500 text-xs truncate"
+          >
+            {{ notificationTypes }}
```

Comment:
Please add the SLA tag for all the notification types.

![CleanShot 2024-03-15 at 12 11 06@2x](https://github.com/chatwoot/chatwoot/assets/12408980/6b51b734-63f9-4614-841d-27e4e683b520)

Reply:
@muhsin-k Done https://github.com/chatwoot/chatwoot/pull/9109/commits/b581d19c93fd757a5c7eee9ce2660af2acbaa61f